process_strava/strava_clean.py

```diff
@@ -0,0 +1,43 @@
+"""A module with helper functions to process data."""
+
+from shapely.geometry import LineString, Point
+import pandas as pd
+
+def swap(x):
```

Comment:
this we can remove. it's for that weird - the data come down in lat, long which is y,x rather than x,y fix.